src/testing/workspace-fixtures.ts

```diff
@@ -87,6 +87,24 @@ export const defaultGoogleWorkspace: GoogleWorkspace = {
   canCompute: true,
 };
 
+export const protectedGoogleWorkspace: GoogleWorkspace = {
+  workspace: {
+    authorizationDomain: [],
+    cloudPlatform: 'Gcp',
+    bucketName: 'fc-secure-00001111-2222-3333-aaaa-bbbbccccdddd',
```

Comment:
so really the only way this differs from the default workspace is the bucket name. I tried a few ways I can think of to replace just the bucket name and reuse the default workspace but couldnt get it to work. any feedback on how to simplify this much appreciated :)

Reply:
I would have expected it to have authorizationDomains. But I think I am probably missing some of the picture here of what it means for a GCP workspace to be "protected".

As far as simplification, you could do this:

`makeGoogleWorkspace({ workspace: { bucketName: 'fc-secure-00001111-2222-3333-aaaa-bbbbccccdddd' } }),`

Reply:
aha! I knew it was something right under my nose :)